app/controllers/summaries_controller.rb

```diff
@@ -0,0 +1,34 @@
+class SummariesController < ApplicationController
+  def new
+    @summary = Summary.new
```

Comment:
Since `Summary` is always connected to a `Post`, I think it will be best here to do:

```ruby
@ post = Post.find(params[:id]) # or params[:post_id] depending on your routes ;-)
@summary = @post.build_summary
```